internal/run.go

```diff
@@ -70,7 +71,14 @@ func strategy() func(*cobra.Command) int {
 		c := make(chan int)
 		go func() {
 			defer close(c)
-			_ = client.NewCollector().Send(cmd)
+			_ = client.NewCollector(util.NewStringSet().Add("err-ignore").
```

Comment:
Ideally, we should generate this list automatically from the flags so that we don't need to remember to update this every time a new flag is added.
I'll open an enhancement issue to remind us.